dss_client/include/dss_client.hpp

```diff
@@ -42,7 +42,7 @@
 namespace dss {
 	namespace py = pybind11;
 
-#define DSS_VER					"20210217"
+#define DSS_VER    "20210217"
```

Comment:
No, I wanted an autogenerated version that other component is doing, we can do it later, not a stopper for merge

Reply:
Fixed as part of build script. The script will get the git tag and overwrite this field